src/main/java/org/openstreetmap/atlas/geography/atlas/packed/README.md

```diff
@@ -1,6 +1,6 @@
 # `PackedAtlas`
 
-This is the base `Atlas`. It is immutable, and stores all its items in large arrays in memory.
+This is the before `Atlas`. It is immutable, and stores all its items in large arrays in memory.
```

Comment:
I like the new `before/after` naming choices over `base/alter`.

Reply:
Cool, I was hoping it was ok.